local_pathfinding/objectives.py

```diff
@@ -37,8 +37,9 @@ def motionCost(self, s1: ob.SE2StateSpace, s2: ob.SE2StateSpace) -> ob.Cost:
 class DistanceObjective(Objective):
     """Generates a distance objective function"""
 
-    def __init__(self, space_information):
+    def __init__(self, space_information, implementation="euclidean"):
```

Comment:
Best way would be to create an enum type for DistanceMethod and TurnCost

For example
```py
from enum import Enum, auto

class Color(Enum):
    RED = auto()
    BLUE = auto()
    GREEN = auto()
```

Resource: https://docs.python.org/3/howto/enum.html

Reply:
Enums can go at the top of this file